helpers/GCResult.ts

```diff
@@ -3,20 +3,20 @@ import { HttpStatusCode, IHttp, ILogger, IRead } from '@rocket.chat/apps-engine/
 import { IApiRequest } from '@rocket.chat/apps-engine/definition/api';
 
 export class GCResults {
-    public atoken: string;
+    public acess_token: string;
```

Comment:
```suggestion
    public access_token: string;
```

Reply:
I mistakenly applied the suggestion; the variable needs to be renamed on the usages as well as in this line.

Reverted for proper alteration

Reply:
@Gautime has this been solved

Reply:
Yeah in this new PR which I will soon open for refresh tokens, I have completely replaced this file with another function. So this variable name will not matter.